Resources/views/Form/form_admin_fields.html.twig

```diff
@@ -166,7 +166,9 @@ file that was distributed with this source code.
         {% endif %}
         <label{% for attrname, attrvalue in label_attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}>
             {{- widget|raw -}}
-            {{- label is not same as(false) ? label|trans({}, translation_domain) -}}
+            {%- if sonata_admin.options['use_icheck'] -%}&nbsp;{%- endif -%}
```

Comment:
yeah css quickly becomes a nightmare to maintain as soon as a project grows...
Things are moving a bit however. We'll soon have variables, maybe nesting...
In the meantime, tooling & conventions...